ydb/library/yql/sql/settings/translation_settings.h

```diff
@@ -125,6 +126,9 @@ namespace NSQLTranslation {
 
         TMaybe<TString> ApplicationName;
         bool PgSortNulls = false;
+
+        NYql::NProto::TTranslationSettings Serialize() const;
+        void UpdateWith(const NYql::NProto::TTranslationSettings& serializedSettings);
```

Comment:
Why `UpdateWith` and not something like `static TTranslationSettings Deserialize(const NYql::NProto::TTranslationSettings& serializedSettings)`?

It matches the way I intend to use serialized settings, i.e. overwrite some fields in the `TTranslationSettings` inherited from the enclosing context.

See usage [here](https://github.com/ydb-platform/ydb/blob/414cd61a450b0e82a9196e7c0dd20fcc555055d5/ydb/library/yql/sql/v1/sql_translation.cpp#L62) and [here](https://github.com/ydb-platform/ydb/blob/414cd61a450b0e82a9196e7c0dd20fcc555055d5/ydb/core/kqp/provider/rewrite_io_utils.cpp#L27).